packages/blade/src/components/Dropdown/DropdownButton.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useDropdown } from './useDropdown';
+import type { ButtonProps } from '~components/Button';
+import { Button } from '~components/Button';
+import { assignWithoutSideEffects } from '~src/utils/assignWithoutSideEffects';
+import { makeAccessible } from '~utils';
+import { getActionListContainerRole } from '~components/ActionList/getA11yRoles';
+
+const _DropdownButton = (props: ButtonProps): JSX.Element => {
+  const {
+    onTriggerClick,
+    onTriggerBlur,
+    onTriggerKeydown,
+    dropdownBaseId,
+    isOpen,
+    activeIndex,
+    hasFooterAction,
+    triggererRef,
+  } = useDropdown();
+  return (
+    <Button
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      ref={triggererRef as any}
+      {...props}
+      {...makeAccessible({
+        hasPopup: getActionListContainerRole(hasFooterAction, 'DropdownButton'),
+        expanded: isOpen,
+        controls: `${dropdownBaseId}-actionlist`,
+        role: 'combobox',
+        activeDescendant: activeIndex >= 0 ? `${dropdownBaseId}-${activeIndex}` : undefined,
+      })}
+      onClick={(e) => {
+        if (props.onClick) {
+          props.onClick(e as React.MouseEvent<HTMLButtonElement>);
+        }
+        onTriggerClick();
+      }}
+      onBlur={() => {
+        // With button trigger, there is no "value" as such. It's just clickable items
+        onTriggerBlur?.({ name: '', value: '' });
```

Comment:
maybe we should return null instead?

Reply:
Changed it to undefined. That we already support in types as standard when values / name doesn't exist. 

If I change it to null here, we have to go and add checks at several places in other inputs to support null everywhere.